src/java/org/apache/cassandra/db/commitlog/CommitLogSegmentManagerCDC.java

```diff
@@ -158,6 +185,16 @@ public void shutdown()
         return alloc;
     }
 
+    // Non-blocking mode is just enabled for CDC. The segment is still marked as FORBIDDEN.
```

Comment:
nit: rephrase to "Non-blocking mode has just recently been enabled for CDC" if that's the intent here. Reads like it's "only" set for CDC which is a little confusing.